source/intel.js

```diff
@@ -338,7 +335,7 @@ const apply_hooks = () => {
 			eventData: data,
 		}]);
 	})
-	NeptunesPride.np.on("confirm_trade_tech", (even, player) => {
+	NeptunesPride.np.on("confirm_trade_tech", (event, player) => {
```

Comment:
Was this an existing bug?

Reply:
Yes, doubt it has any significant impact?

Reply:
Let me try it right now

Reply:
OH wait no. It was an unused variable. But yes it should have been fixed. Ty